index.html

```diff
@@ -3000,44 +3020,94 @@ <h3>Verify Proof</h3>
 
         <p>
 The following algorithm specifies how to check the authenticity and integrity of
-a <a>secured data document</a> by verifying its digital proof. Required inputs
-are a <a>secured data document</a> (<var>securedDocument</var>) and
-<a>proof options</a> (<var>options</var>). The <a>proof options</a> MAY contain
-the following values that help protect against relay and replay attacks:
-<var>expectedProofPurpose</var>, <var>domain</var>, and
-<var>challenge</var>. The <var>expectedProofPurpose</var> value is used to
-ensure that the <var>proof</var> was generated by the proof creator for the
-expected reason by the verifier, such as "authentication". The <var>domain</var>
-value is used by the proof creator to lock a proof to a particular security
-domain, and used by the verifier to ensure that a proof is not being used across
-different security domains. The <var>challenge</var> value is used by the
-verifier to ensure that an attacker is not replaying previously created proofs.
-Additional options are expected to be provided by libraries to ensure that
-digital proof time stamps do not deviate more than an acceptable time frame
-for a given use case. A <dfn>verification result</dfn> is produced as output.
+a <a>secured data document</a> by verifying its digital proof. The algorithm takes as input:
         </p>
 
-        <ol class="algorithm">
+        <ul>
           <li>
-Let <var>proof</var> be set to <var>securedDocument</var>.<var>proof</var>.
+|mediaType|, a [=MIME type=]
           </li>
           <li>
-If the <var>proof</var>.<var>type</var>,
-<var>proof</var>.<var>verificationMethod</var>, or
-<var>proof</var>.<var>proofPurpose</var> values are not set,
+|documentBytes|, a [=byte sequence=] whose media type is |mediaType|
+          </li>
+          <li>
+|expectedProofPurpose|, an optional [=string=], used to ensure that the
+<var>proof</var> was generated by the proof creator for the expected reason by the verifier, such as
+"authentication"
```

Comment:
By convention, credentials use "assertionMethod" and presentations use "authentication"... I think it would be an error for a credential to use authentication, but I am not sure.

Reply:
> I think it would be an error for a credential to use authentication

Yes, that's correct. 

One of the design goals of the Data Integrity work was to separate "authentication" from "asserting claims" as the cryptographic key security around the former is typically different than the cryptographic key security around the latter.

> It'd be nice to have a list of common proof purposes somewhere.

That list can be found here: https://www.w3.org/TR/vc-data-integrity/#proof-purposes

Reply:
Great, thanks!